pkg/apb/provision.go

```diff
@@ -13,7 +13,7 @@ import (
 // Maybe apb defines its own interface and accepts that optionally
```

Comment:
There is no context object supplied during the bind request.  It uses the namespace saved on the service instance during the provision.  This made sense to me since we wouldn't want something different from the actual namespace where the service lives.

https://github.com/cfchase/ansible-service-broker/blob/103d27d9d67e6d44a7aea0d7a9cb94b038cabb2d/pkg/apb/bind.go#L34

https://github.com/cfchase/ansible-service-broker/blob/103d27d9d67e6d44a7aea0d7a9cb94b038cabb2d/pkg/broker/broker.go#L223